opcua/server/address_space.py

```diff
@@ -515,7 +515,7 @@ def set_attribute_value(self, nodeid, attr, value):
 
         for k, v in cbs:
             try:
-                v(k, value)
+                v(k, value, old)
```

Comment:
I am not very keen to send the last value in call back. As you wrote there are several different kind of filters that can be implemented. I think it should be the role of internal subscription to remember last value if it is necessary......